test/frontend.js

```diff
@@ -44,8 +44,52 @@ describe('Frontend', function () {
         .wait(() => document.getElementById('serverMsg').className.indexOf('danger') !== -1)
     })
   })
+  describe('Event Log Test', function () {
+    it('Should Append to the log', async function () {
+      this.timeout(12000)
+      await n.goto(page('event-log'))
+        .should.finally.containEql({ code: 200, url: page('event-log') })
+      let result = await n.wait(4000)
```

Comment:
Well here I don't think it needs to wait for 4 seconds for sure, as the page should be finished loading by this point.